pages/viewPlans/[id].js

```diff
@@ -0,0 +1,54 @@
+import styled from "styled-components"
+import useSWR from "swr"
+import { useRouter } from "next/router"
+import Link from "next/link"
+import WorkoutDay from "@/components/workoutDay/WorkoutDay"
+
+const sharedStyleRules = `
+    width: 100vw;
+`
+
+const PlanContainer = styled.section`
+    border: 1px solid red;
+    height: 100vh;
+`
+
+const PlanHead = styled.section`
```

Comment:
a paragraph makes more sense here:

```suggestion
const PlanHead = styled.p`
```